evap/staff/views.py

```diff
@@ -1476,6 +1463,16 @@ def evaluation_person_management(request, evaluation_id):
             importer_log.forward_messages_to_django(request)
             return redirect("staff:semester_view", evaluation.course.semester.pk)
 
+        #  "test" in operation
```

Comment:
Maybe make it run-time checked?
```suggestion
       assert "test" in operation
```